api/src/test/java/org/openmrs/api/db/ContextDAOTest.java

```diff
@@ -329,4 +342,56 @@ public void should_mergeDefaultRuntimeProperties() {
 		dao.mergeDefaultRuntimeProperties(properties);
 		Assert.assertNotNull(properties.getProperty("hibernate.key"));
 	}
+	
+
+  @Component("testUserSessionListener")
+  public static class TestUserSessionListener implements UserSessionListener {
+    public Set<String> logins = new LinkedHashSet<>();
+    public Set<String> logouts = new LinkedHashSet<>();
+    
```

Comment:
I think it would be nice for the tests to check that the correct Status is passed to the listeners